### DISCUSSION TITLE
使用tkinter插件，簡單在python做出GUI

### DISCUSSION THREAD

**Discussion by Tomlord1122** [1101-GS4719-C/Python-Community-GS4719-C#124] [General]
GUI全名是graphic user interface 使用者介面，大家可能不太清楚這是什麼東西，但其實GUI已經融入在我們的生活中了，
像是我們的windows筆電、mac等等，我們一開機，就會跳到主畫面，點擊應用程式兩下，就會彈出視窗等等

那這邊提供tkinter插件，讓大家能輕鬆用pyhton做出GUI
https://www.youtube.com/watch?v=_wXmXBpCAfc

**Comment by Tomlord1122** [comment]
tkinter 使用上很簡單，能建立一個視窗、按鈕、調節顏色、文字格式等等
連結上課所學的東西，可以簡單的寫一些應用程式出來

**Comment by Jie2222** [comment]
感謝同學分享，感覺簡單容易上手，python套件真強大
